src/app/Components/Artist/ArtistAbout/ArtistAbout.tsx

```diff
@@ -33,7 +33,7 @@ export const ArtistAbout: React.FC<Props> = ({ artist }) => {
   const hasRelatedGenes = relatedGenes.length > 0
 
   return (
-    <Tabs.ScrollView>
+    <Tabs.ScrollView contentContainerStyle={{ paddingHorizontal: 0 }}>
```

Comment:
Do we need to set `paddingHorizontal` to `0` here?

```suggestion
    <Tabs.ScrollView>
```

Reply:
yes, unfortunately. The component comes with a padding. I have a [ticket](https://artsyproduct.atlassian.net/browse/MOPRAT-833) in the practice board to adjust this in mobile and all other screens in Eigen.